README.md

```diff
@@ -38,14 +38,15 @@ Note: you need call the super class `__init__` method with `appetite` equal to 3
 `The hunt began!`
 
 Example:
+
 ```python
 cat = Cat("Cat")
 cat.print_name()  # "Hello, I'm Cat"
 cat.feed()  # "Eating 3 food points"
 
 cat2 = Cat("Cat", False)
 print(cat2.feed())  # 0
-cat2.catch_mouse()  # "The hunt began!"
+catch_mouse()  # "The hunt began!"
 ```
 
 The last class you should implement is a `Dog` class.
```

Comment:
Delete changes that you've made in the `README.md`, this file should not be present in the pull request.

Reply:
not fixed